### ISSUE TITLE
DSMR binding not working with ACE4000GTMM

### ISSUE THREAD

**Issue by Truxen** [openhab/openhab#2103]
see https://groups.google.com/forum/#!category-topic/openhab/dsmr/7qlXt3AZVP0

I belive i have found the problem, why me meter data ist not parsed.

It should work if the REGEX in openhab/bundles/binding/org.openhab.binding.dsmr/src/main/java/org/openhab/binding/dsmr/internal/messages/OBISMsgFactory.java line 37

`^(\\d{1,3}-\\d{1,3}:\\d{1,3}\\.\\d{1,3}\\.\\d{1,3})(\\(.*\\))+$`

 would be changed to:

`^(\\d{1,3}-\\d{1,3}:\\d{1,3}\\.\\d{1,3}\\.\\d{1,3})(\\*\\d{1,3})?(\\(.*\\))+$`

so it would match my OBIS codes:

```
!
/ACE5ACE4KGPRS

0-0:96.1.0*255( XXX )
1-0:1.8.0*255( XXX *kWh)
1-0:1.8.1*255( XXX *kWh)
1-0:2.8.0*255( XXX *kWh)
1-0:2.8.1*255( XXX *kWh)
0-0:96.14.0*255( XXX )
1-0:15.7.0*255( XXX *W)
0-0:17.0.0*255( XXX *A)
0-0:96.3.10*255( XXX )
!
```

**Comment by teichsta** [comment]
@mvolaart could you please take care of this? Best, Thomas E.-E.

**Comment by mvolaart** [comment]
This is a bug. I'm working on this one.

**Comment by teichsta** [comment]
Fix #2158 has been merged into master
